apps/prairielearn/assets/scripts/instructorFileEditorClient.js

```diff
@@ -20,6 +20,7 @@ window.InstructorFileEditor = function (options) {
     showPrintMargin: false,
     mode: options.aceMode,
     readOnly: options.readOnly,
+    enableKeyboardAccessibility: true,
```

Comment:
This editor receives focus on entry. Is this a problem?

Reply:
GitHub's in-browser file editor aut-focuses on page load too; IMO if it's good enough for them, it's good enough for us!